### ISSUE TITLE
rename 'save snapshot' in context menu

### ISSUE THREAD

**Issue by kathryn-baker** [ControlSystemStudio/phoebus#3143]
Following the discussion around save-and-restore at the EPICS Collaboration meeting, would it be possible to rename the 'Save Snapshot' option in the context menu to something like 'Save **Screenshot**'? I think this might help reduce confusion for anyone new to Phoebus and save-and-restore as this 'snapshot' is taking a snapshot of the screen rather than a snapshot of the PV values. 

<img width="185" alt="image" src="https://github.com/user-attachments/assets/175c6ea7-5df6-4ea4-9311-f965988f8862">

**Comment by georgweiss** [comment]
Agree completely

**Comment by kasemir** [comment]
Yes. Actually, "Save Screenshot ...", ending with "..." to indicate that it's not click&done. You'll be prompted for a file name etc.

Just like "Save" (no "...") simply saves, while "Save As..." (with "...") will take a little longer.
Similarly, "Create Log..." and "Send Email ..." need "..." because they have some follow-up interaction.

**Comment by kasemir** [comment]
Thank you for volunteering to clean that up!
